ui/app/components/secret-engine/configure-ssh.hbs

```diff
@@ -3,84 +3,73 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-<form {{on "submit" (perform this.save)}} aria-label="save ssh creds" data-test-configure-form>
-  <div class="box is-fullwidth is-shadowless is-marginless">
-    <NamespaceReminder @mode="save" @noun="configuration" />
-    <MessageError @errorMessage={{this.errorMessage}} />
-    {{#unless @model.isNew}}
-      <p class="has-text-grey-dark">
-        NOTE: You must delete your existing certificate and key before saving new values.
-      </p>
```

Comment:
moved this down, inside the `{{else}}` block
<img width="1709" alt="Screenshot 2024-09-23 at 5 02 31 PM" src="https://github.com/user-attachments/assets/d32a66ff-b098-4075-abfc-243d600e942f">